src/myspace/AdditionalInfo.jsx

```diff
@@ -0,0 +1,141 @@
+        </div>
+        <div className="row">
+          <div className="col-lg-12">
+            <form className="myspace-form mx-auto">
+              <form-group controlId="formUserame">
+                <p className="input-control">
+                  <label htmlFor="username">Username :</label>
+                  <input className="field"
+                    type="text"
+                    name="username"
+                    placeholder={user}
+                    disabled
+                  />
+                </p>
+              </form-group>
+              <div><br></br></div>
+              <form-group controlId="formPhone">
+                <p className="input-control">
+                  <label htmlFor="phone">Phone :</label>
+                  <input className="field"
+                    type="text"
+                    name="phone"
+                    placeholder={additionalInfo.phone}
+                    disabled
+                  />
+                </p>
+              </form-group>
+              <div><br></br></div>
+              <form-group controlId="formMobile">
+                <p className="input-control">
+                  <label htmlFor="mobile">Mobile :</label>
+                  <input className="field"
+                    type="text"
+                    name="mobile"
+                    placeholder={additionalInfo.mobile}
+                    disabled
+                  />
+                </p>
+              </form-group>
+              <div><br></br></div>
+              <form-group controlId="formPersonalWebsite">
+                <p className="input-control">
+                  <label htmlFor="personalWebsite">Personal Website :</label>
+                  <input className="field"
+                    type="text"
+                    name="personalWebsite"
+                    placeholder={additionalInfo.personal_website}
+                    disabled
+                  />
+                </p>
+              </form-group>
+              <div><br></br></div>
+              <form-group controlId="formTimezone">
+                <p className="input-control">
+                  <label htmlFor="timezone">Timezone :</label>
+                  <input className="field"
+                    type="text"
+                    name="timezone"
+                    placeholder={TIMEZONE_ENUM[additionalInfo.timezone]}
+                    disabled
+                  />
+                </p>
+              </form-group>
+              <div><br></br></div>
+              <form-group controlId="formIsOrganizationRepCheck">
+                <div className="row">
+                  <div className="col-sm">
+                    <p className="input-control">
+                      <input
+                        type="checkbox"
+                        name="is_organization_rep"
+                        aria-label="is_organization_rep"
+                        defaultChecked={additionalInfo.is_organization_rep}
```

Comment:
are we using snake case or camel case? 

additionalInfo = camel case
is_organization_rep = snake case

Reply:
snake case on the response body of backend as per python convention. do you want the frontend to be changed as well (like in useState we use camel case)

Reply:
@meenakshi-dhanani . I've replaced the snake case in any other place except setting value from dict additionalInfo as this is as per what being sent by backend (as python format)